backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/DefaultAndroidInput.java

```diff
@@ -419,7 +420,7 @@ public void processEvents () {
 						processor.mouseMoved(e.x, e.y);
 						break;
 					case TouchEvent.TOUCH_SCROLLED:
-						processor.scrolled(e.scrollAmount);
+					    processor.scrolled(e.scrollAmountX,e.scrollAmountY);
```

Comment:
There is a space missing between `e.scrollAmountX,` and `e.scrollAmountY`.

Reply:
fixed